### PR TITLE
Enabled automatic label adding for MM PRs

### PR DESCRIPTION
**Pull request by melinath**
Closes b/295224144 - follow-up to https://github.com/GoogleCloudPlatform/magic-modules/pull/9390

```release-note:none

```

### CODE REVIEW

File: .ci/scripts/go-plus/github-differ/generate_comment.sh
Comment (trodge) [inline .ci/scripts/go-plus/github-differ/generate_comment.sh:1]: Would it be possible to also make these changes to the new generate comment command in the magician? It is located here: https://github.com/GoogleCloudPlatform/magic-modules/blob/main/.ci/magician/cmd/generate_comment.go

I'm currently working on making the test for this command more flexible and more specific to each function it uses.

The plan is to replace this script with a call to the new magician command.

If changing the code in both places in this PR is inconvenient, please leave a TODO in `generate_comment.go` and I will look into it next week.

**Review by trodge — APPROVED** [review]
Overall LGTM, just a question about making these changes in the new magician command.

File: .ci/scripts/go-plus/github-differ/generate_comment.sh
Comment (melinath) [reply .ci/scripts/go-plus/github-differ/generate_comment.sh:1]: I'm not sure how to test the go version apart from running unit tests - is that possible / necessary at this point or will you be doing testing before launching?

Comment (trodge) [reply .ci/scripts/go-plus/github-differ/generate_comment.sh:1]: I'll be running it manually before launching. Let's go ahead and merge this PR and I'll re-base off it.

**Review by trodge — APPROVED** [review]

**Pull request merged**